### PR TITLE
workaround.ch

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by J0WI** [comment]
See also:
bitflu.workaround.ch
slackintosh.workaround.ch

File: src/chrome/content/rules/workaround.ch.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!-- bitflu. ERR_CONNECTION_REFUSED
```
Comment (J0WI) [inline src/chrome/content/rules/workaround.ch.xml]: please use keyword refused

File: src/chrome/content/rules/workaround.ch.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+slackintosh. NET::ERR_CERT_AUTHORITY_INVALID -->
```
Comment (J0WI) [inline src/chrome/content/rules/workaround.ch.xml]: please use keyword self-signed

**Comment by galeksandrp** [comment]
Added.

**Pull request merged**